### ISSUE TITLE
train without pretrain weights。。。

### ISSUE THREAD

**Issue by AiIsBetter** [DetectionTeamUCAS/Cascade_FPN_Tensorflow#5]
I don't want to use the weight of the pre-training to train the model, where can I change；
I modified ：
            # restorer, restore_ckpt = faster_rcnn.get_restorer()
            # if not restorer is None:
            #     restorer.restore(sess, restore_ckpt)
            #     print('restore model')
added:
            summary_writer = tf.summary.FileWriter(summary_path, graph=sess.graph)
            saver = tf.train.Saver(max_to_keep=10)
But the result is not correct, no target can be predicted, and then the loss is also down.I look forward to your reply. Thank you. @yangxue0827 .

**Comment by zhao34731** [comment]
Also I train from scratch, but the loss is nan for VOC dataset.  Here is an example , I wonder whether the loss will be normal after training a long time. I chck the anchor sizes  and stride. It seems fine for the voc dataset. 

step160    image_name:b'000321.jpg' |	
                                  rpn_loc_loss:0.005 |	 rpn_cla_loss:0.364 |	 rpn_total_loss:0.369 |
                                  fast_rcnn_loc_loss:0.000 |	 fast_rcnn_cla_loss:1.953 |	 fast_rcnn_total_loss:1.953 |
                                  total_loss:nan |	 per_cost_time:0.227s
